templates/accordion.jsx

```diff
@@ -0,0 +1,104 @@
+import Adapt from 'core/js/adapt';
+import { compile, classes, templates, html } from 'core/js/reactHelpers';
+
+export default function (model, view) {
+  const data = model.toJSON();
+  data._globals = Adapt.course.get('_globals');
+  return (
+    <div className="component__inner accordion__inner">
+
+      {templates.component(model, view)}
+
+      <div className="component__widget accordion__widget">
+
+        {data._items.map(({ _graphic, _classes, title, body, _index, _isVisited, _isActive }, index) =>
+
+          <div
+            className={classes([
+              'accordion__item',
+              _graphic.src && 'has-image',
+              _classes && _classes
+            ])}
+            key={_index}
+            data-index={_index}
+          >
+
+            <button
+              id={`${data._id}-${index}-accordion-button`}
+              className={classes([
+                'accordion__item-btn',
+                'js-toggle-item',
+                _isVisited && 'is-visited',
+                _isActive ? 'is-open is-selected' : 'is-closed'
+              ])}
+              onClick={view.onClick.bind(view)}
+              aria-expanded={_isActive ? 'true' : 'false'}
```

Comment:
Is `_isActive` a Boolean?
```suggestion
              aria-expanded={_isActive.toString()}
```